code/game/objects/items/granters.dm

```diff
@@ -523,6 +526,32 @@
 		desc = "It's a damaged upgrade module."
 		name = "damaged board"
 
+/obj/item/book/granter/martial/worldshaker
+	martial = /datum/martial_art/worldshaker
+	name = "Prototype worldshaker compound"
```

Comment:
capitalization is different than other martial arts (this one + the IPC one but that one has version one which i think is supposed to be a name so)
```suggestion
	name = "prototype worldshaker compound"
```